packages/ui/src/components/select/select.tsx

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { cx } from "classix";
+import React, { ComponentProps } from "react";
+import {
+  Button as BaseButton,
+  FieldError as BaseFieldError,
+  Form as BaseForm,
+  Label as BaseLabel,
+  ListBox as BaseListBox,
+  ListBoxItem as BaseListBoxItem,
+  Popover as BasePopover,
+  Select as BaseSelect,
+  SelectValue as BaseSelectValue,
+} from "react-aria-components";
+
+export const Form = (properties: ComponentProps<typeof BaseForm> & { className?: string | undefined }) => {
```

Comment:
I don't think we want tackle forms quite yet. For example, the `w-full` class is specific to these stories.